crates/oxc_transformer/src/lib.rs

```diff
@@ -43,9 +45,12 @@ use crate::{
 
 pub struct Transformer<'a> {
     ctx: Ctx<'a>,
+
     // NOTE: all callbacks must run in order.
     x0_typescript: TypeScript<'a>,
     x1_react: React<'a>,
+    // x2_decorators,
+    x3_es2022: Es2022<'a>,
```

Comment:
Ordering here is weird. When we add es2024, es2025, etc, all the numbers will shift. We probably need a different ordering system.